.azure-pipelines/all.yml

```diff
@@ -1,137 +1,137 @@
 schedules:
-- cron: 0 5 * * *
-  displayName: Nightly (EDT/EST)
-  always: true
-  branches:
-    include:
-    - master
+  - cron: 0 5 * * *
+    displayName: Nightly (EDT/EST)
+    always: true
+    branches:
+      include:
+        - master
```

Comment:
We should avoid making all of these whitespace changes to the file ..

Reply:
Agreed.  That's not something I'd normally do, so I'm guessing that my text editor was being "helpful".  Fortunately `yamllint` says it's clean.